src/ui/Facet/FacetSearchElement.ts

```diff
@@ -9,6 +9,7 @@ import { FacetSearchUserInputHandler } from './FacetSearchUserInputHandler';
 import { uniqueId } from 'underscore';
 import { ISearchDropdownNavigator, ISearchDropdownConfig } from './FacetSearchDropdownNavigation/DefaultSearchDropdownNavigator';
 import { SearchDropdownNavigatorFactory } from './FacetSearchDropdownNavigation/SearchDropdownNavigatorFactory';
+import { KEYBOARD } from '../../Core';
```

Comment:
This should be imported from `KeyboardUtils` instead of from `Core`.
Less side effect.